src/components/Thanks/ShareChallenge.vue

```diff
@@ -194,7 +194,7 @@ export default {
 				return sum + (value?.amountLent ?? 0);
 			}, 0) ?? 0;
 
-			return (fundedAmount / targetAmount) * 100 || 0;
+			return Math.round((fundedAmount / targetAmount) * 100) || 0;
```

Comment:
This won't work in the 99.8% situation. Please add a check to round down in that situation, if we keep rounding in.